README.md

```diff
@@ -1,6 +1,16 @@
 # Bevy Quickstart
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io). You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
+The template brought to you by the Bevy Jam working group.
```

Comment:
I get including this, but from the perspective of a user, this isn't the most useful line to start with. Also `The` -> `This`.

Reply:
[`8f5dc8e` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/8f5dc8e6d0b0fb81f36841d11c62d6cdcf13f9df) better?

Reply:
I started a new comment so I'll resolve this one.